assets/src/edit-story/karma/fixture.js

```diff
@@ -0,0 +1,376 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import React, { useCallback, useState, useMemo, forwardRef } from 'react';
+import { render, act } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import App from '../app/index';
+import APIProvider from '../app/api/apiProvider';
+import APIContext from '../app/api/context';
+import { TEXT_ELEMENT_DEFAULT_FONT } from '../app/font/defaultFonts';
+import Layout from '../app/layout';
+import FixtureEvents from './fixtureEvents';
+
+const DEFAULT_CONFIG = {
+  storyId: 1,
+  api: {},
+  allowedMimeTypes: {
+    image: ['image/png', 'image/jpeg', 'image/jpg', 'image/gif'],
+    video: ['video/mp4'],
+  },
+  allowedFileTypes: ['png', 'jpeg', 'jpg', 'gif', 'mp4'],
+  capabilities: {},
+};
+
+export class Fixture {
+  constructor() {
+    this._config = { ...DEFAULT_CONFIG };
```

Comment:
Can we make this configurable in the constructor?

Reply:
We will be adding a bunch of configuration methods that should be called before `render()` is called. But I'd rather stay away from the constructor args.